__tests__/app.test.js

```diff
@@ -45,3 +45,42 @@ describe("GET/api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200 - returns an object containing the correct information based on the article_id provided by the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const articleObj = body.article;
```

Comment:
great testing.  but you shuold be sending back a single article object rather than an article nested in an  array